lib/Mojo/Util.pm

```diff
@@ -56,13 +55,16 @@ my %CACHE;
 
 our @EXPORT_OK = (
   qw(b64_decode b64_encode camelize class_to_file class_to_path decamelize),
-  qw(decode deprecated dumper encode files getopt hmac_sha1_sum html_unescape),
+  qw(decode deprecated dumper encode getopt hmac_sha1_sum html_unescape),
   qw(md5_bytes md5_sum monkey_patch punycode_decode punycode_encode quote),
-  qw(secure_compare sha1_bytes sha1_sum slurp split_cookie_header),
-  qw(split_header spurt steady_time tablify term_escape trim unindent unquote),
-  qw(url_escape url_unescape xml_escape xor_encode)
```

Comment:
Should the functions be removed from EXPORT_OK, before they are removed from this module?